java/ingest/ingest-core/src/main/java/sleeper/ingest/job/IngestJobValidationUtils.java

```diff
@@ -37,16 +42,20 @@ public static <T> Optional<T> deserialiseAndValidate(
         T job;
         try {
             job = deserialiser.apply(message);
+            LOGGER.info("Deserialised message to ingest job {}", job);
         } catch (RuntimeException e) {
+            LOGGER.warn("Deserialisation failed: {}", e.getStackTrace());
```

Comment:
With SLF4J you log out a stack trace like this: `LOGGER.warn("Message", e);`

All the logging methods have a version that takes a Throwable as the last parameter, and you don't need the curly brackets in the message.

Reply:
Maybe it would also be worth logging out the full message in this case, like this:

`LOGGER.warn("Deserialisation failed for message: {}", message, e);`